src/plugins/ABTestingService.js

```diff
@@ -29,6 +29,13 @@ const ABTestingService = {
     ABTesting.submit = ABTesting.submit || { index: 1 }
     ABTesting.submit.deleted = true
 
+    ABTesting.parentCelibataire = ABTesting.parentCelibataire || { index: 1 }
+    ABTesting.parentCelibataire.value =
+      ABTesting.parentCelibataire.value || Math.random() >= 0.5
+        ? "afficheCelibataire"
+        : null
```

Comment:
Je ne sais pas comment Matomo gère le _null_. J'aurais tendance à mettre une valeur plutôt que _null_ pour éviter de ne pas pouvoir accéder aux stats de cette variante-là. _ceinture, bretelles, à l'aveugle_ :sweat_smile: 
```suggestion
        : "cacheCelibaire"
```